grabbit/src/integrationTest/groovy/com/twcable/grabbit/server/services/ServerServiceSpec.groovy

```diff
@@ -72,7 +72,7 @@ class ServerServiceSpec extends Specification {
 
         when:
         //This will also actually execute the Batch Job internally
-        syncServerService.getContentForRootPath("/default.groovy", (Collection<String>)Collections.EMPTY_LIST, "", mockServletOutputStream)
+        syncServerService.getContentForRootPath("", "/default.groovy", (Collection<String>)Collections.EMPTY_LIST, "", mockServletOutputStream)
```

Comment:
@sagarsane I must admin this looks a little odd to me.  I see you added a new param for this function so the first one should be username, but I'm not sure what is going on here.  I assume this test is passing, but I'm not entirely sure why :wink: